modules/react/action-bar/lib/ActionBar.tsx

```diff
@@ -1,76 +1,55 @@
 import * as React from 'react';
-import {styled} from '@workday/canvas-kit-react/common';
-import {colors, commonColors, space, CSSProperties} from '@workday/canvas-kit-react/tokens';
+import {createComponent, styled, StyledType} from '@workday/canvas-kit-react/common';
+import {commonColors, colors, space, depth} from '@workday/canvas-kit-react/tokens';
+import {Globals} from 'csstype';
 
-export interface ActionBarProps extends React.HTMLAttributes<HTMLDivElement> {
+type PropertyPosition =
+  | Globals
+  | '-webkit-sticky'
+  | 'absolute'
+  | 'fixed'
+  | 'relative'
+  | 'static'
+  | 'sticky';
+
+export type ActionBarProps = StyledType & {
```

Comment:
Move the `StyledType` to line 23 instead. `createComponent` adds the `as` type. Having `StyledType` might interfere with the `as` provided by `createComponent`